.github/workflows/tests.yml

```
@@ -16,7 +16,7 @@ jobs:
       - uses: actions/setup-python@v5
         with:
           python-version: "3.10"
-      - uses: pre-commit/action@v3.0.0
+      - uses: pre-commit/action@v3.0.1
```

```suggestion
      - uses: pre-commit/action@v3
```
don't use a specific patch version to avoid this kind of bump